### PR TITLE
Fix #394 by stripping the module name from the table name

### LINKED ISSUE #394: Sqlite3 Exception when database name contains a colon
The following code produces a table with the name `cwiki::user`

```crystal
module Cwiki
  class User < Granite::Base
  end
end
```

This is an issue as statements that reference the table look something like this: "SELECT * FROM cwiki::user". Sqlite throws errors on table names that contain a colon unless the name is surrounded by quotation marks.

I think this should be an easy fix, but I'm not familiar with this codebase. I'll fork and check it out and report back if I figure things out.

### PR DESCRIPTION
**Pull request by 3n-k1**
Fixes #394

### CODE REVIEW

**Comment by Blacksmoke16** [comment]
Nice!  A spec to go with it would be :100:.

**Comment by 3n-k1** [comment]
I'll be honest, I'm pretty new to crystal and have no idea how to write a spec. Could you point me in the direction of a tutorial for it?

**Comment by Blacksmoke16** [comment]
@3n-k1 https://crystal-lang.org/reference/guides/testing.html would be a good start.

After reading that, define a new model type within: https://github.com/amberframework/granite/blob/master/spec/spec_models.cr

Something like
```cr
class Foo::Bar::Model < Granite::Model
  column id : Int64, primary: true
end
```

The spec would be like:
```cr
it "strips the namespace when defining the default table now" do
  Foo::Bar::Model.table_name.should eq "model"
end
```

**Comment by 3n-k1** [comment]
Great, thank you very much!

**Comment by Blacksmoke16** [comment]
@3n-k1 Sorry, `Granite::Base` not `Granite::Model` my bad 😆

**Comment by 3n-k1** [comment]
No problem at all :)

**Comment by Blacksmoke16** [comment]
Ogh, apparently there is already a `Foo` model somewhere.

Lets go with: `MyApp::Namespace::Model`.

**Comment by 3n-k1** [comment]
It's a good thing you're here, I'd never be able to debug this haha

File: spec/granite/table/table_spec.cr [hunk 1]
```diff
@@ -9,6 +9,10 @@ describe Granite::Table do
     it "sets the table name based on class name if not specified" do
       SongThread.table_name.should eq "song_thread"
     end
+
+    it "strips the namespace when defining the default table now" do
+      Foo::Bar::Model.table_name.should eq "model"
```
Comment (Blacksmoke16) [inline spec/granite/table/table_spec.cr:14]:
```suggestion
      MyApp::Namespace::Model.table_name.should eq "model"
```

Comment (3n-k1) [reply spec/granite/table/table_spec.cr:14]: Whoops, sorry, missed that

**Comment by 3n-k1** [comment]
Running the tests locally this time to be safe

**Comment by Blacksmoke16** [comment]
Looks like specs are still failing.  I'll take a look.

**Comment by 3n-k1** [comment]
That's strange, since they run fine for me in docker. I appreciate you looking into it.

**Comment by Blacksmoke16** [comment]
Sorry for the delay, imma check this out today.

File: spec/spec_models.cr [hunk 2]
```diff
@@ -509,6 +509,10 @@ end
     column my_enum : MyEnum?, column_type: "TEXT", converter: Granite::Converters::Enum(MyEnum, String)
   end
 
+  class MyApp::Namespace::Model < Granite::Base
+    column id : Int64, primary: true
+  end
```
Comment (Blacksmoke16) [inline spec/spec_models.cr:514]:
```suggestion
  class MyApp::Namespace::Model < Granite::Base
    connection {{ adapter_literal }}
  
    column id : Int64, primary: true
  end
```

**Comment by 3n-k1** [comment]
Ah, can't believe I missed that. Thanks for the merge!

**Pull request merged**